lib/cli.js

```diff
@@ -25,6 +25,7 @@ const cli = meow({
           -s, --skip-unused     Skip check for unused packages.
           -p, --production      Skip devDependencies.
           -E, --save-exact      Save exact version (x.y.z) instead of caret (^x.y.z) in package.json.
+          -i, --installer <npm> Change npm installer. (eg 'pnpm', 'ied')
```

Comment:
:+1:  `NPM_CHECK_INSTALLER` sounds good, changes will be minimal.
i will write the new PR.